### PR TITLE
Allow resizing of the point drawing tool

### LINKED ISSUE #2157: Point marking task: smaller mark size for fine marking
Solar Stormwatch would like to use a marking task with a smaller mark size, for marking the fine lines on their jmap images. Could be implemented by adding the point size as an option to the drawing task eg. select from small, medium or large points. Default to large for backward compatibility with existing projects.

### PR DESCRIPTION
**Pull request by eatyourgreens**
Adds an option to the point tool to allow the size of the points to be specified via a dropdown menu in the project builder.

Closes #2157

### CODE REVIEW

**Comment by eatyourgreens** [comment]
Staged at https://point-tool-resize.pfe-preview.zooniverse.org/

**Comment by srallen** [comment]
We may want to consider moving the delete button to a different position when the point is small. The delete button takes a significant area of the point itself. However, functionally, this is working cross-browser. 👍

**Comment by brian-c** [comment]
I agree about moving the delete button. Otherwise, looks good.

File: app/classifier/tasks/drawing/markings-renderer.cjsx [hunk 1]
```diff
@@ -64,6 +64,7 @@ module.exports = React.createClass
                 mark: mark
                 details: toolDescription.details
                 color: toolDescription.color
+                size: toolDescription.size ? 'large'
```
Comment (brian-c) [inline app/classifier/tasks/drawing/markings-renderer.cjsx]: Can we move the default into the tool's `getDefaultProps`?

Comment (eatyourgreens) [inline app/classifier/tasks/drawing/markings-renderer.cjsx]: Good point. I've moved it into the point tool.

**Comment by eatyourgreens** [comment]
Feedback from Chris Scott:
>Right. The points are a good size now but the ‘x’ icon of the selected point makes it tricky to adjust once placed (although easy to delete!).

Definitely need to move the delete button.

**Comment by eatyourgreens** [comment]
OK, I've moved the delete button out to the same radius as large points, and bumped the selected size a little too. Changes should be deployed to https://point-tool-resize.pfe-preview.zooniverse.org/

**Comment by srallen** [comment]
LGTM

**Pull request merged**